src/custom-element.js

```diff
@@ -219,7 +219,11 @@ export function copyElementToChildWindow(parentWin, childWin, name) {
  */
 export function upgradeElementInChildWindow(parentWin, childWin, name) {
   const toClass = getExtendedElements(parentWin)[name];
-  dev().assert(toClass, '%s is not stubbed yet', name);
+  // Some extensions unofficially register unstubbed elements, e.g. amp-bind.
```

Comment:
Extensions stub an element of the same name, but `amp-bind` registers the `amp-state` element which won't be stubbed. That's okay since `amp-state` is really just metadata, but we shouldn't `assert()` here for such elements. #9143 will make this distinction clearer.